### ISSUE TITLE
Project+

### ISSUE THREAD

**Issue by Meester-Moo** [webmastersmith/compTIA_Search#1]
Hello, I haven't actually used this yet, but I was wondering if this works for Project+. Just started the WGU class today.

**Comment by webmastersmith** [comment]
Yes, It works. You can follow the directions and build your own, or use my pre-built executable. I have not updated them since I took the class, so some page links are broken. Enjoy!
